lua/metals.lua

```diff
@@ -182,4 +188,32 @@ M['metals/status'] = function(_, _, params)
   end
 end
 
+M['metals/publishDecorations'] = function(err, _, decorations)
+  if err then
+    print("metals.publishDecorations: Server error:", err)
```

Comment:
We probably don't want to print the entire error do we? I think it may be best to just let the user know there is an error, and then let them to a `:MetalsToggleLogs` to get a closer look at what's going on. I'm just afraid the `err` here may not be then nicest thing to show there. What do you think?

Reply:
Yes, unifying the error mgt is pending, I don't know what are the best practices in nvim. In this case it's definitely worth removing the print as it prints the stackoverflow.

Reply:
Sounds good 👍